app/actions/GroupActions.js

```diff
@@ -205,3 +219,49 @@ export function fetchMembershipsPagination({
     );
   };
 }
+
+export function createInterestGroup(group: Object): Thunk<*> {
```

Comment:
I think create and remove can also be made generic for groups, and we can just add type as a parameter. (Or you could add a small wrapper with the same args as the old `createInterestGroup` that just sets `type = 'interesse'`.